<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TrialBalance" time="53" tests="1" failures="0" errors="1">
   <testsuite name="TrialBalance" tests="1" failures="0" errors="1" time="53" timestamp="2019-05-14 14:47:22" hostname="dipak - DIPAKDAS-PC.indusnettechnologies.com" id="Test Suites/TrialBalance">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="E:\\MarkitCore\\MarkitCore-Final\\MarkitCoreSystem\\MarkitCore\\Reports\\TrialBalance\\20190514_144715"/>
         <property name="logFiles" value="E:\\MarkitCore\\MarkitCore-Final\\MarkitCoreSystem\\MarkitCore\\Reports\\TrialBalance\\20190514_144715\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="dipak - DIPAKDAS-PC.indusnettechnologies.com"/>
         <property name="os" value="Windows 7 32bit"/>
         <property name="katalonVersion" value="6.0.5.4"/>
         <property name="browser" value="Chrome 74.0.3729.131"/>
         <property name="hostAddress" value="10.0.12.237"/>
         <property name="sessionId" value="1a48c716d5c2df8f7c70729076120bbf"/>
         <property name="seleniumVersion" value="3.7.1"/>
         <property name="proxyInformation" value="ProxyInformation{proxyOption=NO_PROXY, proxyServerType=HTTP, password=, proxyServerAddress=, proxyServerPort=0}"/>
         <property name="platform" value="Windows 7"/>
      </properties>
      <testcase name="Test Cases/TC01-LoginApplication" classname="Test Cases/TC01-LoginApplication" status="PASSED">
         <system-out><![CDATA[2019-05-14 14:47:30 - [TEST_CASE][PASSED] - Test Cases/TC01-LoginApplication: Test Cases/TC01-LoginApplication

2019-05-14 14:47:31 - [TEST_STEP][NOT_RUN] - openBrowser(""): NOT_RUN: openBrowser("")

2019-05-14 14:47:31 - [MESSAGE][NOT_RUN] - NOT_RUN: openBrowser("")

2019-05-14 14:47:31 - [TEST_STEP][NOT_RUN] - navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display"): NOT_RUN: navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display")

2019-05-14 14:47:31 - [MESSAGE][NOT_RUN] - NOT_RUN: navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display")

2019-05-14 14:47:31 - [TEST_STEP][PASSED] - setText(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username"), username): Text 'dDasTester' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username'

2019-05-14 14:47:31 - [MESSAGE][PASSED] - Text 'dDasTester' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username'

2019-05-14 14:47:31 - [TEST_STEP][PASSED] - setText(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password"), password): Text 'Czybm6R*' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password'

2019-05-14 14:47:31 - [MESSAGE][PASSED] - Text 'Czybm6R*' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password'

2019-05-14 14:47:31 - [TEST_STEP][PASSED] - delay(4, STOP_ON_FAILURE): Delayed 4 second(s)

2019-05-14 14:47:35 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-14 14:47:35 - [TEST_STEP][NOT_RUN] - maximizeWindow(): NOT_RUN: maximizeWindow()

2019-05-14 14:47:35 - [MESSAGE][NOT_RUN] - NOT_RUN: maximizeWindow()

2019-05-14 14:47:35 - [TEST_STEP][PASSED] - click(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login")): Object: 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login' is clicked on

2019-05-14 14:47:41 - [MESSAGE][PASSED] - Object: 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login' is clicked on

2019-05-14 14:47:41 - [TEST_STEP][NOT_RUN] - log = new com.kms.katalon.core.logging.KeywordLogger(): NOT_RUN: log = new com.kms.katalon.core.logging.KeywordLogger()

2019-05-14 14:47:41 - [MESSAGE][NOT_RUN] - NOT_RUN: log = new com.kms.katalon.core.logging.KeywordLogger()

2019-05-14 14:47:41 - [TEST_STEP][NOT_RUN] - log.logInfo(“Dipak”): NOT_RUN: log.logInfo(“Dipak”)

2019-05-14 14:47:41 - [MESSAGE][NOT_RUN] - NOT_RUN: log.logInfo(“Dipak”)]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/TC12-TrialBalance-Insurer" classname="Test Cases/TC12-TrialBalance-Insurer" status="ERROR">
         <error type="ERROR" message="Test Cases/TC12-TrialBalance-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more
"/>
         <system-out><![CDATA[2019-05-14 14:47:41 - [TEST_CASE][ERROR] - Test Cases/TC12-TrialBalance-Insurer: Test Cases/TC12-TrialBalance-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more


2019-05-14 14:47:41 - [TEST_STEP][PASSED] - waitForJQueryLoad(5): jQuery is ready.

2019-05-14 14:47:42 - [MESSAGE][PASSED] - jQuery is ready.

2019-05-14 14:47:42 - [TEST_STEP][PASSED] - scrollToElement(findTestObject("TC11-TrialBalance/Report"), 4): Scroll to object 'Object Repository/TC11-TrialBalance/Report' successfully

2019-05-14 14:47:42 - [MESSAGE][PASSED] - Scroll to object 'Object Repository/TC11-TrialBalance/Report' successfully

2019-05-14 14:47:42 - [TEST_STEP][PASSED] - click(findTestObject("TC11-TrialBalance/Report")): Object: 'Object Repository/TC11-TrialBalance/Report' is clicked on

2019-05-14 14:47:42 - [MESSAGE][PASSED] - Object: 'Object Repository/TC11-TrialBalance/Report' is clicked on

2019-05-14 14:47:42 - [TEST_STEP][PASSED] - delay(4): Delayed 4 second(s)

2019-05-14 14:47:46 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-14 14:47:46 - [TEST_STEP][PASSED] - scrollToElement(findTestObject("TC11-TrialBalance/TrialBalance"), 4): Scroll to object 'Object Repository/TC11-TrialBalance/TrialBalance' successfully

2019-05-14 14:47:46 - [MESSAGE][PASSED] - Scroll to object 'Object Repository/TC11-TrialBalance/TrialBalance' successfully

2019-05-14 14:47:46 - [TEST_STEP][PASSED] - click(findTestObject("TC11-TrialBalance/TrialBalance")): Object: 'Object Repository/TC11-TrialBalance/TrialBalance' is clicked on

2019-05-14 14:47:48 - [MESSAGE][PASSED] - Object: 'Object Repository/TC11-TrialBalance/TrialBalance' is clicked on

2019-05-14 14:47:48 - [TEST_STEP][PASSED] - waitForJQueryLoad(4): jQuery is ready.

2019-05-14 14:47:48 - [MESSAGE][PASSED] - jQuery is ready.

2019-05-14 14:47:48 - [TEST_STEP][PASSED] - selectOptionByLabel(findTestObject("TC11-TrialBalance/SelectCurrency"), "USD", false): All options with label 'USD' are selected in object 'Object Repository/TC11-TrialBalance/SelectCurrency' 

2019-05-14 14:47:49 - [MESSAGE][PASSED] - All options with label 'USD' are selected in object 'Object Repository/TC11-TrialBalance/SelectCurrency' 

2019-05-14 14:47:49 - [TEST_STEP][PASSED] - delay(4): Delayed 4 second(s)

2019-05-14 14:47:53 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-14 14:47:53 - [TEST_STEP][PASSED] - click(findTestObject("TC11-TrialBalance/SelectEntity")): Object: 'Object Repository/TC11-TrialBalance/SelectEntity' is clicked on

2019-05-14 14:47:53 - [MESSAGE][PASSED] - Object: 'Object Repository/TC11-TrialBalance/SelectEntity' is clicked on

2019-05-14 14:47:53 - [TEST_STEP][PASSED] - delay(4): Delayed 4 second(s)

2019-05-14 14:47:57 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-14 14:47:57 - [TEST_STEP][PASSED] - setText(findTestObject("TC11-TrialBalance/TypeEntityName"), "New Insurer"): Text 'New Insurer' is set on object 'Object Repository/TC11-TrialBalance/TypeEntityName'

2019-05-14 14:47:57 - [MESSAGE][PASSED] - Text 'New Insurer' is set on object 'Object Repository/TC11-TrialBalance/TypeEntityName'

2019-05-14 14:47:57 - [TEST_STEP][PASSED] - sendKeys(findTestObject("TC11-TrialBalance/TypeEntityName"), Keys.chord(ENTER)): Keys 'ENTER + ' sent to object: 'Object Repository/TC11-TrialBalance/TypeEntityName'

2019-05-14 14:47:57 - [MESSAGE][PASSED] - Keys 'ENTER + ' sent to object: 'Object Repository/TC11-TrialBalance/TypeEntityName'

2019-05-14 14:47:57 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-14 14:48:02 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-14 14:48:02 - [TEST_STEP][PASSED] - scrollToElement(findTestObject("TC12-TrialBalance-Insurer/InsurerCreditor"), 4): Scroll to object 'Object Repository/TC12-TrialBalance-Insurer/InsurerCreditor' successfully

2019-05-14 14:48:02 - [MESSAGE][PASSED] - Scroll to object 'Object Repository/TC12-TrialBalance-Insurer/InsurerCreditor' successfully

2019-05-14 14:48:02 - [TEST_STEP][PASSED] - click(findTestObject("TC12-TrialBalance-Insurer/InsurerCreditor")): Object: 'Object Repository/TC12-TrialBalance-Insurer/InsurerCreditor' is clicked on

2019-05-14 14:48:03 - [MESSAGE][PASSED] - Object: 'Object Repository/TC12-TrialBalance-Insurer/InsurerCreditor' is clicked on

2019-05-14 14:48:03 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-14 14:48:08 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-14 14:48:08 - [TEST_STEP][PASSED] - setText(findTestObject("TC11-TrialBalance/SearchPolicy"), findTestData("TestData/MTA_PolicyNO").getValue(1, 1)): Text 'DASD/POLICY/003095' is set on object 'Object Repository/TC11-TrialBalance/SearchPolicy'

2019-05-14 14:48:09 - [MESSAGE][PASSED] - Text 'DASD/POLICY/003095' is set on object 'Object Repository/TC11-TrialBalance/SearchPolicy'

2019-05-14 14:48:09 - [TEST_STEP][PASSED] - sendKeys(findTestObject("TC11-TrialBalance/SearchPolicy"), Keys.chord(ENTER)): Keys 'ENTER + ' sent to object: 'Object Repository/TC11-TrialBalance/SearchPolicy'

2019-05-14 14:48:09 - [MESSAGE][PASSED] - Keys 'ENTER + ' sent to object: 'Object Repository/TC11-TrialBalance/SearchPolicy'

2019-05-14 14:48:09 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-14 14:48:14 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-14 14:48:14 - [TEST_STEP][PASSED] - InsurerComm = getText(findTestObject("TC12-TrialBalance-Insurer/InsurerValueCapture")): Text of object 'Object Repository/TC12-TrialBalance-Insurer/InsurerValueCapture' is: '84.00'

2019-05-14 14:48:15 - [MESSAGE][PASSED] - Text of object 'Object Repository/TC12-TrialBalance-Insurer/InsurerValueCapture' is: '84.00'

2019-05-14 14:48:15 - [TEST_STEP][ERROR] - com.excel.helper.ExcelHelper.updateTheExcel("Data Files\TestData\Comission.xlsx", "InsurerCommission", InsurerComm): com.excel.helper.ExcelHelper.updateTheExcel("Data Files\TestData\Comission.xlsx", "InsurerCommission", InsurerComm) FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more


2019-05-14 14:48:15 - [MESSAGE][ERROR] - java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)

2019-05-14 14:48:15 - [MESSAGE][ERROR] - com.excel.helper.ExcelHelper.updateTheExcel("Data Files\TestData\Comission.xlsx", "InsurerCommission", InsurerComm) FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more]]></system-out>
         <system-err><![CDATA[2019-05-14 14:47:41 - [TEST_CASE][ERROR] - Test Cases/TC12-TrialBalance-Insurer: Test Cases/TC12-TrialBalance-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more]]></system-err>
      </testcase>
      <system-out><![CDATA[2019-05-14 14:47:22 - [TEST_SUITE][ERROR] - TrialBalance: Test Cases/TC12-TrialBalance-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more]]></system-out>
      <system-err><![CDATA[2019-05-14 14:47:22 - [TEST_SUITE][ERROR] - TrialBalance: Test Cases/TC12-TrialBalance-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC12-TrialBalance-Insurer.run(TC12-TrialBalance-Insurer:59)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557825435791.run(TempTestSuite1557825435791.groovy:35)
Caused by: java.io.FileNotFoundException: Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1556793110169.run(Script1556793110169.groovy:59)
	... 13 more]]></system-err>
   </testsuite>
</testsuites>
